mmrazor/models/architectures/dynamic_ops/bricks/dynamic_conv.py

```diff
@@ -37,16 +37,21 @@ def __init__(self, *args, **kwargs) -> None:
     def convert_from(cls, module: nn.Conv2d) -> 'DynamicConv2d':
         """Convert an instance of nn.Conv2d to a new instance of
         DynamicConv2d."""
-        return cls(
-            in_channels=module.in_channels,
-            out_channels=module.out_channels,
-            kernel_size=module.kernel_size,
-            stride=module.stride,
-            padding=module.padding,
-            dilation=module.dilation,
-            groups=module.groups,
-            bias=True if module.bias is not None else False,
-            padding_mode=module.padding_mode)
+        # a group-wise conv will not be converted to dynamic conv
+        if module.groups > 1 and not (module.groups == module.out_channels ==
```

Comment:
Add a warning.
Remind users that xx（module name) is a group-wise conv and will not be pruned

Reply:
It's difficult to add a warning with the module name in method 'convert_from', as there is not accessible to module name. If add warning out of 'convert_from', there may be too many places to add warnings. As a result, I add the warning in 'convert_from' without module name.